### DISCUSSION TITLE
Информация о плейлистах, в которые включен трек

### DISCUSSION THREAD

**Discussion by EldarAliev** [Chimildic/YaMuTools#88] [Идея]
Предлагаю для рассмотрения к реализация фичу (если реально реализовать такое).
Вывод информации о плейлистах, в которые добавлен выбранный трек у пользователя. Например в "Мне нравится", в "В машину", "В метро" и т.д.
Добавить эту информацию между блоками "О треке" и "Похожие". Пример на скриншоте.
![image](https://github.com/Chimildic/YaMuTools/assets/49719359/295235f7-6265-45b6-ae62-f4878912527e)

**Comment by kisoid** [comment]
Это уже есть в ЯМузыке стандартно.

![изображение](https://github.com/Chimildic/YaMuTools/assets/97474829/75a23647-4ab5-4e4d-8230-ff8840f7603c)

**Comment by EldarAliev** [comment]
Да, это видел, но когда список плейлистов больше сотни (судя по скрину у Вас так же), то найти эту заветную галочку, в этом микро-окне, которое постоянно при прокрутке ещё исчезает, не очень удобно. А когда выбрал трек и видно не только инфо о нём и т.д., но ещё и плейлисты, где он уже пользователем добавлен будет гораздо быстрее и удобнее.
